activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -93,6 +93,17 @@ def column_exists?(table_name, column_name, type = nil, options = {})
                                       (!options.key?(:null)      || c.null == options[:null]) }
       end
 
+      # Returns just a table's primary key
+      def primary_key(table_name)
+        pks = primary_keys(table_name)
+        return nil unless pks.size == 1
+        pks.first
```

Comment:
Current behavior of `primary_key` method is the following:

* return single column if the table has single column primary key
* return `nil` if the table has composite primary key beause of #18070, #18259.
* return `nil` if the table has no primary key.

This PR keep current behavior for now.

Anyway, I'll fix to Rubyish conditionals.